hack/update-repository-to-cpe.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+cd "$(git rev-parse --show-toplevel)"
+
+BASE_URL='https://access.redhat.com/security/data/meta/v1/repository-to-cpe.json'
+
+# Update this variable for any additional repos that should be added to the list. This is a JSON
+# formatted list, e.g.
+#   export EXTRAS='[
+#       "foo",
+#       "bar",
+#   ]'
+# Having a comma on the last item is not necessarily valid JSON, but yq handles it well.
+export EXTRAS='[
+]'
+
+export COMMENT='
+This file is automatically generated by hack/update-repository-to-cpe.sh. Do not update it directly.
+'
+
+curl -L "${BASE_URL}" | \
+    yq '.data |
+        [to_entries[].key] as $repos |
+        $repos + env(EXTRAS) | sort as $repos |
+        {"known_rpm_repositories": $repos} |
+        . head_comment=env(COMMENT)' > data/repository_to_cpe.yaml
```

Comment:
Maybe a different filename, e.g.:

```suggestion
        . head_comment=env(COMMENT)' > data/known_rpm_repositories.yaml
```

Reply:
Also the name of the script.

Reply:
Good eye. Forgot to update those from an earlier iteration.